build/winobjc.version

```diff
@@ -0,0 +1,14 @@
+#define VERSION_MAJOR               0
```

Comment:
Not quite. For now the workflow for changing the tools will be:
- Make changes
- Check in changes, including the re-built binaries, into git. Do nothing with the version.

The monthly release workflow is:
- Update the winobjc.version file with a new release string
- Build the SDK and publish the zip file.

---
In reply to: [84107821](https://github.com/Microsoft/WinObjC/pull/1184#discussion_r84107821) [](ancestors = 84107821)